### ISSUE TITLE
[BUG] - Lidarr - Fixed output name but more than one file to download

### ISSUE THREAD

**Issue by ObscenityIB** [RandomNinjaAtk/arr-scripts#334]
**Application**
Lidarr

**Host platform**
Docker Compose

**Script**
Video

**Script Version**
3.9

**Describe the bug**
Videos fail to download via yt-dlp with the log:
```
lidarr  | 2024-12-03 10:06:53 :: Video :: 3.9 :: 2/63 :: Blood on the Dance Floor :: IMVDB :: Processing 5 Videos!
lidarr  | 2024-12-03 10:07:00 :: Video :: 3.9 :: 2/63 :: Blood on the Dance Floor :: IMVDB :: 2/5 :: Bewitched :: https://www.youtube.com/watch?v=4bEDM4bsfWg...
lidarr  | ERROR: Fixed output name but more than one file to download: /config/extended/downloads/videos/incomplete/Bewitched-video
lidarr  | 2024-12-03 10:07:00 :: Video :: 3.9 :: 2/63 :: Blood on the Dance Floor :: IMVDB :: 2/5 :: Bewitched :: Download failed, skipping...
```

**To Reproduce**
1. Set Video Path.
2. Wait for Video script to run.

**Comment by ObscenityIB** [comment]
> Did you put a cookie.txt in your config folder? It seems to trigger this bug for me

Yep, though it seems to remove all the contents and replace it with its own.

**Comment by CpHeat** [comment]
> > Did you put a cookie.txt in your config folder? It seems to trigger this bug for me
> 
> Yep, though it seems to remove all the contents and replace it with its own.
> 
> Edit: Removing it seems to be a workaround for now. Now all I need is to figure out the copyright errors and the private video errors, like why would imvdb list private and blocked videos as a source? Super weird. Age Verification won't be possible until the cookies file is usable.

I think at least in some cases that the videos went private or blocked after being added to imvdb and I don’t think they can do much about it. One issue I have is imvdb seems like a dying site, it has been unavailable for multiple days this week, and haven’t reviewed multiple entries I added a month ago, and even big artists page are really badly outdated (like multiple years of music videos missing)
